### ISSUE TITLE
Optimal Value of Exp_reg_weight (for explainability mask loss) 

### ISSUE THREAD

**Issue by gunshi** [tinghuiz/SfMLearner#60]
Hey, I wanted to inquire as to what value was used to weigh the explainability mask loss (cross entropy with reference mask), since in the code the default value is set to 0 (exp_reg_weight).
Thanks!

**Comment by tinghuiz** [comment]
I used 0.2 for the CVPR paper. But there has been some change to the model (e.g. batch norm is removed), and I haven't got time to verify if 0.2 still works.
